Formula/mesheryctl.rb

```diff
@@ -0,0 +1,32 @@
+class Mesheryctl < Formula
+  desc "Command-line utility for Meshery, the cloud native management plane"
+  homepage "https://meshery.io"
+  url "https://github.com/meshery/meshery.git",
+      tag:      "v0.6.0-rc.6b",
+      revision: "bb80f4a97c8020aae6624c672ddd2a07e84c2733"
+  version "0.6.0-rc.6b"
```

Comment:
This needs to wait for a stable version: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
In `mehsery` we make releases of this kind in our `stable` release channel and would want our users to be able to use these versions. 

/cc @leecalcote (if you'd like to add anything)

Reply:
Does `0.6.0-rc.6b` not mean "0.6.0 Release Candidate 6b"? Because if it does, that does not sound like a stable version to me.

Reply:
@SMillerDev, yes, this version is part of our stable release channel.